commands/dbaas/postgres/completer/ids.go

```diff
@@ -51,17 +56,18 @@ func ClustersIds() []string {
 	if err != nil {
 		return nil
 	}
-	ids := make([]string, 0)
-	if dataOk, ok := clusterList.ClusterList.GetItemsOk(); ok && dataOk != nil {
-		for _, item := range *dataOk {
-			if itemId, ok := item.GetIdOk(); ok && itemId != nil {
-				ids = append(ids, *itemId)
-			}
-		}
-	} else {
+
+	convertedClusterList, err := json2table.ConvertJSONToTable(
+		"items", jsonpaths.DbaasPostgresCluster, clusterList.ClusterList,
+	)
+	if err != nil {
 		return nil
 	}
-	return ids
+
+	return completions.NewCompleter(convertedClusterList, "ClusterId").AddInfo("DisplayName").AddInfo(
+		"Location",
+		"(%v)",
```

Comment:
what would this `(%v)` print?

Reply:
sounds good, was just an unusual function call and wasn't sure of the expected output